lib/tasks/decidim_vocdoni_wepacker_tasks.rake

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "decidim/gem_manager"
+
+namespace :decidim_vocdoni do
+  namespace :webpacker do
+    desc "Installs Vocodoni webpacker files in Rails instance application"
+    task install: :environment do
+      raise "Decidim gem is not installed" if decidim_path.nil?
+
+      install_vocdoni_npm
+    end
+
+    desc "Adds Vocodoni dependencies in package.json"
+    task upgrade: :environment do
+      raise "Decidim gem is not installed" if decidim_path.nil?
+
+      install_vocdoni_npm
+    end
+
+    def install_vocdoni_npm
+      vocdoni_npm_dependencies.each do |type, packages|
+        puts "install NPM packages. You can also do this manually with this command:"
+        puts "npm i --save-#{type} #{packages.join(" ")}"
+        system! "npm i --save-#{type} #{packages.join(" ")}"
+      end
+    end
+
+    def vocdoni_npm_dependencies
+      @vocdoni_npm_dependencies ||= begin
+        package_json = JSON.parse(File.read(vocdoni_path.join("package.json")))
+
+        {
+          prod: package_json["dependencies"].map { |package, version| "#{package}@#{version}" },
+          dev: package_json["devDependencies"].map { |package, version| "#{package}@#{version}" }
```

Comment:
While running this task I had the following exception:

```shell
$ bin/rails decidim_vocdoni:webpacker:install

rails aborted!
NoMethodError: undefined method `map' for nil:NilClass
/home/apereira/Work/vocdoni/demo-vocdoni-decidim/lib/tasks/decidim_vocdoni_wepacker_tasks.rake:35:in `vocdoni_npm_dependencies'
/home/apereira/Work/vocdoni/demo-vocdoni-decidim/lib/tasks/decidim_vocdoni_wepacker_tasks.rake:22:in `install_vocdoni_npm'
/home/apereira/Work/vocdoni/demo-vocdoni-decidim/lib/tasks/decidim_vocdoni_wepacker_tasks.rake:11:in `block (3 levels) in <main>'
/home/apereira/Work/vocdoni/demo-vocdoni-decidim/bin/rails:5:in `<top (required)>'
/home/apereira/Work/vocdoni/demo-vocdoni-decidim/bin/spring:10:in `block in <top (required)>'
/home/apereira/Work/vocdoni/demo-vocdoni-decidim/bin/spring:7:in `<top (required)>'
Tasks: TOP => decidim_vocdoni:webpacker:install
(See full trace by running task with --trace)
```

(Mind that I copied this file to my app just to try this out). 

This is because in this line we're trying to use the `devDependencies` key and in our case we still didn't have any. As a quick workaround I added jest as a development dependency, even though we're not using it yet, but I wanted to add at least some basic support as we've much JS code. 

(Just mentioning that we probably would want to change this method and check that `devDependencies` is indeed defined)

Reply:
well, there will be lots of `devDependencies` once this is merged no?

Reply:
Oh you're right, I missed that, sorry!